app/views/favorites/of_user.ctp

```diff
@@ -43,45 +55,56 @@ $numberOfSentences = count($favorites);
 <div id="main_content">
     <div class="module">
     
-    <h2><?php echo $title . ' ('. $numberOfSentences.')'; ?></h2>
-    
     <?php
-    
-    if ($numberOfSentences > 0) {
-        $type = 'mainSentence';
-        $parentId = null;
-        $withAudio = false;
-        $ownerName = null;
-        foreach ($favorites as $favorite) {
-            if (empty($favorite['Sentence']['text'])) {
-                $sentenceId = $favorite['Favorite']['favorite_id'];
-                $linkToSentence = $html->link(
-                    '#'.$sentenceId,
-                    array(
-                        'controller' => 'sentences',
-                        'action' => 'show',
-                        $sentenceId
-                    )
-                );
+    if ($userExists === false) {
+        $commonModules->displayNoSuchUser($username, $backLink);
+    } else {
+        echo '<h2>';
+        echo $title . ' ('. $numberOfSentences.')';
```

Comment:
We have a more or less standard way to set titles of paginated results. Take other paginated pages as example, such as [that one](https://github.com/Tatoeba/tatoeba2/blob/dev/app/views/transcriptions/of.ctp#L50-L54).